### DISCUSSION TITLE
After passing parameters into st.produce_scattertext_explorer function open.write function throws html error

### DISCUSSION THREAD

**Discussion by SuryaV21** [explosion/spaCy#11505] [Help: Coding & Implementations] [answered]
Hello Team,

I am newbie to scattertext package and I really want to plot scatter chart using this. So I started using this and passed data frame. When I tried to show the chart using open and html it throws error stating "name 'html' is not defined" or "Write function cannot take module".

Request you to help me in understanding and resolving the issue.

**Comment by polm** [comment, marked as answer]
Can you give an example of your code?

It sounds like you're trying to use scattertext with Streamlit. But note that while scattertext uses spaCy (and is in the Universe for that reason), we don't develop it, so it might be better to ask for help directly at the [scattertext](https://github.com/JasonKessler/scattertext) repo or Streamlit forums.

**Comment by SuryaV21** [comment]
Thank you Polm. Closing the thread.
